configuration/configuration.go

```diff
@@ -277,42 +278,56 @@ type FileChecker struct {
 	Threshold int `yaml:"threshold,omitempty"`
 }
 
-// Redis configures the redis pool available to the registry webapp.
 type Redis struct {
-	// Addr specifies the the redis instance available to the application.
-	Addr string `yaml:"addr,omitempty"`
+	// Either a single address or a seed list of host:port addresses
+	// of cluster/sentinel nodes.
+	Addrs []string
 
-	// Usernames can be used as a finer-grained permission control since the introduction of the redis 6.0.
-	Username string `yaml:"username,omitempty"`
+	// ClientName will execute the `CLIENT SETNAME ClientName` command for each conn.
+	ClientName string
 
-	// Password string to use when making a connection.
-	Password string `yaml:"password,omitempty"`
+	// Database to be selected after connecting to the server.
+	// Only single-node and failover clients.
+	DB int
 
-	// DB specifies the database to connect to on the redis instance.
-	DB int `yaml:"db,omitempty"`
+	Protocol         int
```

Comment:
What does `Protocol` stand for?

Reply:
> Protocol 2 or 3. Use the version to negotiate RESP version with redis-server. Default is 3.

It seems to have been removed in the latest v9.

Reply:
Probably would be useful to carry the comments from here https://github.com/redis/go-redis/blob/v9.1.0/options.go#L31-L139 into this struct as well in case questions like these come up again.